src/Spinners/__tests__/LoadSpinner.test.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { render } from 'test-utils';
+import LoadSpinner from '../LoadSpinner';
+
+describe('Testing Data Loading Spinner', function () {
```

Comment:
Super minor point, but our convention is for the top-level describe block to just be the name of the component (e.g. `LoadSpinner`). Just makes for cleaner test output.

Reply:
ok